### PR TITLE
[MM-18518] Enhanced status: webapp and status command

### PR DESCRIPTION
**Pull request by gabrieljackson**
This change does the following:
 - Adds `cloud status` command for displaying all clusters and
   installations on the configured cloud server.
 - Adds webapp functionality in the form of displaying user's
   installations in their profile popups.
 - Minor logging and bug fixes.

https://mattermost.atlassian.net/browse/MM-18518

### CODE REVIEW

File: server/api.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		p.handleProfileImage(w, r)
+	case "/api/v1/userinstalls":
+		p.handleUserInstalls(w, r)
+	default:
+		http.NotFound(w, r)
+	}
+}
+
+func (p *Plugin) handleUserInstalls(w http.ResponseWriter, r *http.Request) {
+	p.API.LogWarn(fmt.Sprintf("%+v", r.Header))
```
Comment (jwilander) [inline server/api.go]: Debug message?

File: server/api.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	userID := r.Header.Get("Mattermost-User-ID")
+	if userID == "" {
+		http.Error(w, "Not authorized", http.StatusUnauthorized)
+		return
+	}
+
+	installsForUser, err := p.getUpdatedInstallsForUser(userID)
+	if err != nil {
+		http.Error(w, "something happened", http.StatusForbidden)
```
Comment (jwilander) [inline server/api.go]: Would a 500 be more accurate?

File: webapp/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "github",
```
Comment (jwilander) [inline webapp/package.json]: These fields need updating

File: webapp/src/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+class Plugin {
+    async initialize(registry) {
+        registry.registerReducer(Reducer);
+
+        registry.registerPopoverUserAttributesComponent(UserAttribute);
+    }
+
+    deinitialize() {
+        //eslint-disable-next-line no-console
+        console.log('Shutting down cloud plugin');
```
Comment (jwilander) [inline webapp/src/index.js]: Debugging message?

File: server/api.go (hunk 1 above)
Comment (gabrieljackson) [reply server/api.go]: Yep, thanks for catching that. I thought I had cleaned them all up.

File: server/api.go (hunk 2 above)
Comment (gabrieljackson) [reply server/api.go]: Also, another placeholder error message I forgot about. Thanks.

File: webapp/src/index.js (hunk 4 above)
Comment (gabrieljackson) [reply webapp/src/index.js]: This is actually nearly identical to what the demo plugin does. I don't know how useful it is to keep or not.

Comment (jwilander) [reply webapp/src/index.js]: I think that's just in the demo plugin to show that there is a deinit function that can be used

**Pull request merged**